code/index.html

```diff
@@ -2,17 +2,83 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project Name</title>
+    <title>Kreativhang</title>
     <link rel="stylesheet" href="./style.css" />
     <!-- dont forget the meta tag for responsiveness -->
   </head>
   <body>
     <header>
       <!-- If you want a header with a title, logo and a navbar, here is where you should add that. -->
+      <div class="hamburger">
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
+
+      <nav>
+        <a href="./postkarten.html">Postkarten</a>
+        <a href="./faltkarten.html">Faltkarten</a>
+        <a href="./stickers.html">Stickers</a>
+        <a href="./geschenkpapier.html">Geschenkpapier</a>
+      </nav>
+    
+      <img src="img/Kreativhang_Logo.svg" alt="Kreativhang"/>
+
+      <h1>Freude schenken</h1>
+
     </header>
 
+
     <main class="">
       <!-- The main container to use as the flex parent for your card layout, the main content of the site should be in here.  -->
-    </main>
+   
+      <section class="flex-container">
+          <section class="container">
+              <div class="karten">
+                <img src="img/Postkarte_1.jpg" alt="Postkarte Nostalgie 6390"/>
+                <h2>Postkarte Nostalgie 6390</h2>
+                <h3>CHF 2.50</h3>
+              </div>
+
+              <div class="karten">
+                <img src="img/Postkarte_2.jpg" alt="Postkarte Nostalgie Aufstieg"/>
+                <h2>Postkarte Nostalgie Aufstieg</h2>
+                <h3>CHF 2.50</h3>
+              </div>
+
+              <div class="karten">
+                <img src="img/Postkarte_3.jpg" alt="Postkarte Nostalgie Bitzistock"/>
+                <h2>Postkarte Nostalgie Bitzistock</h2>
+                <h3>CHF 2.50</h3>
+              </div>
+
+              <div class="karten">
+                <img src="img/Postkarte_4.jpg" alt="Postkarte Nostalgie Brunni"/>
+                <h2>Postkarte Nostalgie Brunni</h2>
+                <h3>CHF 2.50</h3>
+              </div>
+
+              <div class="karten">
+                <img src="img/Postkarte_5.jpg" alt="Postkarte Nostalgie Entdecken"/>
+                <h2>Postkarte Nostalgie Entdecken</h2>
+                <h3>CHF 2.50</h3>
+              </div>
+ 
+              <div class="karten">
+                <img src="img/Postkarte_6.jpg" alt="Postkarte Nostalgie Abenteuer"/>
+                <h2>Postkarte Nostalgie Abenteuer</h2>
+                <h3>CHF 2.50</h3>
+              </div>
+          </section>           
+      </section>
+   
+    </main> 
```

Comment:
This closing `</main>` tag doesn't have a corresponding opening tag at the top: `<main>`